app/controllers/pieces_controller.rb

```diff
@@ -17,6 +31,8 @@ def promote_pawn
     old_piece = pawn.color + '_' + pawn.type.downcase
     new_piece = pawn.game.pieces.create(pawn_promotion_params.merge(color: pawn.color, row_coordinate: pawn.row_coordinate, column_coordinate: pawn.column_coordinate, has_moved?: true))
     pawn.destroy
+    game_channel = 'game_channel-' + new_piece.game.id.to_s
```

Comment:
Since we've used this twice. Could be extracted to a method on `game`

game.rb

```
def web_socket_channel
  "game_channel-#{ id }"
end
```